modules/contact/src/main/kotlin/edu/stanford/spezi/modules/contact/component/AddressCard.kt

```diff
@@ -66,9 +72,12 @@ fun NavigationCard(address: String, publisher: (OnAction) -> Unit) {
 
 @Composable
 @Preview
```

Comment:
Just a small project-specific thing. We have created the annotation `@ThemePreviews`, which always adds a preview of both dark and light mode. 

And if we only display small components in the preview and the whole screen is possibly obstructive, you can set the following so that you only see the component in the preview:
`SpeziTheme(isPreview = true) {`

Reply:
I changed the views to make use of this now - thank you!